app/src/app.js

```diff
@@ -5,6 +5,27 @@ const app = express();
 const bodyParser = require('body-parser');
 app.use(bodyParser.json());
 
+app.get('/posts/:id', async (req, res) => {
+
+    try {
+        if (isNaN(req.params.id)) {
+            res.status(400).json({
+                message: 'id must be a number'
+            });
+            return;
+        }
+        let post = await postModel.getPost(req.params.id);
+        if (post === null) {
+            res.status(404);
```

Comment:
brak `end`, odpowiedź nie zostanie wysłana

Reply:
poprawione.

Reply:
no to jeszcze dołóż `return` ;P

Reply:
gotowe :-)